docs/en/ingest-management/elastic-agent/view-elastic-agent-status.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[view-elastic-agent-status]]
+[role="xpack"]
+= View status of {agent}s
+
+To view the status of your {fleet}-managed agents, go to
+*Management > Fleet > Agents*.
+
+[role="screenshot"]
+image::images/agent-status.png[Agents tab showing status of each {agent}]
+
+{fleet} reports these statuses:
+
+|=== 
+
+| *Healthy* | The {agent} is online and running as expected. There are no agent
+policy updates or automatic agent binary updates pending, but there might be
```

Comment:
There are no agent policy updates or automatic agent binary updates ** in progress**